lib/gym/runner.rb

```diff
@@ -1,6 +1,7 @@
 require 'pty'
 require 'open3'
 require 'fileutils'
+require 'shellwords'
```

Comment:
Hmmm, excellent question.  Other fastlane tools (fastlane, sign, snapshot) have .rb files that similarly require 'shellwords', but do not declare it in the gemspec.